### PR TITLE
Series of AI85 Jira Ticket Fixes.

### PR DESCRIPTION
**Pull request by sihyung-maxim**

### CODE REVIEW

File: Libraries/PeriphDrivers/Source/I2C/i2c_me17.c [hunk 1]
```diff
@@ -48,7 +48,10 @@
 #include "i2c_reva.h"
 
 /* **** Definitions **** */
+#define MXC_I2C_STD_MODE 100000
+#define MXC_I2C_FAST_SPEED 400000
 #define MXC_I2C_FASTPLUS_SPEED 1000000
+#define MXC_I2C_HS_MODE 3400000
```
Comment (Jacob-Scheiffler) [inline Libraries/PeriphDrivers/Source/I2C/i2c_me17.c:54]: I think these suffixes should all match. Either all end in "_SPEED" or "_MODE", dealer's choice.

File: Libraries/PeriphDrivers/Source/I2C/i2c_reva.c [hunk 2]
```diff
@@ -144,15 +144,18 @@ int MXC_I2C_RevA_SetFrequency(mxc_i2c_reva_regs_t *i2c, unsigned int hz)
...
+    if (hz > MXC_I2C_REVA_FASTPLUS_SPEED && hz <= MXC_I2C_REVA_HS_MODE) {
+        // Enable high speed mode
         int hsLowClks, hsHiClks;
 
         // Calculate the period of SCL and set up 33% duty cycle
         ticksTotal = PeripheralClock / hz;
         hsLowClks = (ticksTotal * 2) / 3 - 1;
         hsHiClks = ticksTotal / 3 - 1;
 
+        printf("Periph: %d\nHz: %d\n", PeripheralClock, hz);
```
Comment (Jacob-Scheiffler) [inline Libraries/PeriphDrivers/Source/I2C/i2c_reva.c:156]: Why print? Is this leftover from debug?

File: Libraries/PeriphDrivers/Source/UART/uart_me17.c [hunk 3]
```diff
@@ -263,13 +263,15 @@ int MXC_UART_SetFlowCtrl(mxc_uart_regs_t *uart, mxc_uart_flow_t flowCtrl, int rt
             MXC_GPIO_Config(&gpio_cfg_uart0_flow);
             break;
 
+#if (TARGET_NUM != 78000)
```
Comment (Jacob-Scheiffler) [inline Libraries/PeriphDrivers/Source/UART/uart_me17.c:266]: Are these necessary? Looking at max78000_files.mk, I don't see uart_me17.c getting added to the list of files to compile.

**Review by Jacob-Scheiffler — CHANGES_REQUESTED** [review]

File: Libraries/PeriphDrivers/Source/I2C/i2c_me17.c (hunk 1 above)
Comment (Jake-Carter) [reply Libraries/PeriphDrivers/Source/I2C/i2c_me17.c:54]: +1 for `_SPEED`

File: Libraries/PeriphDrivers/Source/UART/uart_me17.c (hunk 3 above)
Comment (sihyung-maxim) [reply Libraries/PeriphDrivers/Source/UART/uart_me17.c]: Nope.

File: Libraries/PeriphDrivers/Source/I2C/i2c_reva.c (hunk 2 above)
Comment (sihyung-maxim) [reply Libraries/PeriphDrivers/Source/I2C/i2c_reva.c]: Yep.

File: Libraries/PeriphDrivers/Source/I2C/i2c_me17.c (hunk 1 above)
Comment (sihyung-maxim) [reply Libraries/PeriphDrivers/Source/I2C/i2c_me17.c]: Renamed to HIGH_SPEED

**Review by Jacob-Scheiffler — APPROVED** [review]

**Pull request merged**